recipe/meta.yaml

```
@@ -9,17 +9,18 @@ source:
   sha256: 593526acae1c0fda0ea6c48439f67c3943094c542fe769f8b90fe9e6c6cc4871
 
 build:
-  number: 1
+  number: 2
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   skip: true  # [py<37]
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython >=0.29.32                        # [build_platform != target_platform]
+    - cython >=0.29.32                       # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
+    - setuptools_scm[toml] ~=7.0.0
```

Conda doesn't understand the extras syntax in Python. We need to be explicitly about the optional dependencies.

```suggestion
    - setuptools_scm ~=7.0.0
    # setuptools_scm[toml]
    - setuptools >=42
    
```

---

yeah, that's easy enough to do, but doesn't help the issue it seems. That [requirement already was satisfied](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=605883&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642&l=124)

---

Fun! Some more context: https://github.com/statsmodels/statsmodels/issues/8497 and https://twitter.com/ocefpaf/status/1562400506437373953

---

Move setuptools_scm to host.

---

I made these changes.

---

https://github.com/statsmodels/statsmodels/issues/8497 is unrelated to this.  That was about package managers like poetry that want to know deps without downloading whls.